SunshineGameFinder/Program.cs

```diff
@@ -1,10 +1,17 @@
 // See https://aka.ms/new-console-template for more information
+using Gameloop.Vdf;
+using Gameloop.Vdf.Linq;
 using Newtonsoft.Json;
 using SunshineGameFinder;
 using System.CommandLine;
+using System.Text.RegularExpressions;
+
+// constants
+const string wildcatDrive = @"*:\";
+const string steamLibraryFolders = @"Program Files (x86)\Steam\steamapps\libraryfolders.vdf";
 
 // default values
-var gameDirs = new List<string>() { @"C:\Program Files (x86)\Steam\steamapps\common", @"C:\XboxGames", @"C:\Program Files\EA Games" };
+var gameDirs = new List<string>() { @"*:\Program Files (x86)\Steam\steamapps\common", @"*:\XboxGames", @"*:\Program Files\EA Games" };
```

Comment:
Not that it's a problem with your PR, but I just realized I didn't put Epic Games in if you wanted to go ahead and add that 
```*:\Program Files\Epic Games\```